src/components/TaskStatusIconLegend/index.tsx

```diff
@@ -0,0 +1,94 @@
+import { useState } from "react";
+import styled from "@emotion/styled";
+import IconButton from "@leafygreen-ui/icon-button";
+import Popover from "@leafygreen-ui/popover";
+import { spacing } from "@leafygreen-ui/tokens";
+import { Disclaimer, Overline } from "@leafygreen-ui/typography";
+import Icon from "components/Icon";
+import { PopoverContainer } from "components/styles/Popover";
+import { groupedIconStatuses } from "components/TaskStatusIcon";
+import { taskStatusToCopy } from "constants/task";
+import { size, zIndex } from "constants/tokens";
+
+export const LegendContent = () => (
+  <>
+    <TitleContainer>
+      <Overline>NEW ICONS LEGEND</Overline>
+      <IconButton aria-label="Close Task Status Icon Legend">
+        <Icon glyph="X" />
+      </IconButton>
+    </TitleContainer>
+    <Container>
+      {groupedIconStatuses.map(({ icon, statuses }) => {
+        const label = statuses.map((status) => (
+          <Disclaimer key={status}>{taskStatusToCopy[status]}</Disclaimer>
+        ));
+        return (
+          <Row key={statuses.join()}>
+            {icon}
+            <LabelContainer>{label}</LabelContainer>
+          </Row>
+        );
+      })}
+    </Container>
+  </>
+);
+
+export const TaskStatusIconLegend: React.FC = () => {
+  const [isActive, setIsActive] = useState(false);
+
+  return (
+    <IconButton
+      onClick={() => {
+        setIsActive(!isActive);
+      }}
```

Comment:
Looks like the click listener is acting on the tooltip. When i click anywhere on the tooltip it closes and when i click outside it does not. I would expect the opposite behavior it should only close when we click outside the tooltip.

Reply:
I think some users may want to reference the legend as they interact between the waterfall and history pages so I decided to exclude the outside click functionality. I think clicking anywhere on the popover to close makes sense because there are no other click events in the tooltip and it becomes easier to close. Although the design guidelines section isn't filled out, the popover on [mongodb.design](https://www.mongodb.design/component/popover/example/) behaves this way. Let me know what you think

Reply:
Personally, I'm not a fan of the default leafygreen `Popover` behavior... both `mui` and `antd` popovers don't close when clicking on the popover content, which is what I prefer. This doesn't have to be done in this ticket but maybe we should consider creating a modified leafygreen `Popover` component in the future, since this issue has come up a few times